lib/streamlit/elements/selectbox.py

```diff
@@ -13,9 +13,9 @@
 # limitations under the License.
 
 from textwrap import dedent
-from typing import Any, Callable, Optional, cast
+from typing import TYPE_CHECKING, Any, Callable, Optional, Sequence, TypeVar, cast
+from typing import overload
```

Comment:
nit: these two import statements can be merged

Reply:
I hope that when we introduce isort, such problems will be detected and fixed automatically.

Reply:
Yeah, agreed. It's been suggested previously, and I believe no one is in opposition to adding it as part of our build process.

My last attempt at introducing it unfortunately didn't work out, though, as I only had a few hours of free time at the time and ran into some issues that I wasn't able to quickly resolve

Reply:
isort would be helpful. Might look into it if I feel inspired one day. For now, the imports are merged.